api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.13.0"
+   "version": "v1.12.0"
```

Comment:
I think v1.13 here should not be decreased to 1.12.

`Maybe` this mumber is related to `git tag`.

```
git describe 
v1.13.0-alpha.0-299-gf8f220b49b
```